qiskit/transpiler/preset_passmanagers/level0.py

```diff
@@ -163,14 +163,20 @@ def _swap_mapped(property_set):
         pre_opt += translation
     else:
         pre_opt = None
-    if scheduling_method is None or scheduling_method in {"alap", "asap"}:
+
+    if scheduling_method is None:
         sched = common.generate_scheduling(
-            instruction_durations, scheduling_method, timing_constraints, inst_map, target=target
+            instruction_durations,
+            scheduling_method,
+            timing_constraints,
+            inst_map,
+            target=target,
```

Comment:
I'm wondering if it would be better to go about this the opposite way, add a third plugin with the entrypoint `default` that sets `scheduling_method=None` when calling `generate_scheduling()` during `pass_manager()`? This would remove the need for an if else logic in all the optimization levels and end up with less code (assuming you did `scheduling_method = pass_manager_config.scheduling_method or "default"`). What do you think?

Reply:
that makes sense, then we can fully use the plugin stuff